import/trello/importTrello.ts

```diff
@@ -9,7 +9,8 @@ import {IPropertyOption, IPropertyTemplate, createBoard} from '../../webapp/src/
 import {createBoardView} from '../../webapp/src/blocks/boardView'
 import {createCard} from '../../webapp/src/blocks/card'
 import {createTextBlock} from '../../webapp/src/blocks/textBlock'
-import {Trello} from './trello'
+import {createCheckboxBlock, CheckboxBlock} from '../../webapp/src/blocks/checkboxBlock'
```

Comment:
ESlint called out that `CheckboxBlock` and `ChecklistElement` aren't used, and should not be imported.

Reply:
Yeah my editor says the same, not sure why they're still there to be honest, must've had a language server error when I tidied and committed. Had to set up the TypeScript/JavaScript environment from scratch for this.